transcendence-app/src/shared/db/postgres/postgresConnection.provider.ts

```diff
@@ -0,0 +1,26 @@
+import { Pool } from 'pg';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class PostgresPool {
+  private pool: Pool;
+
+  constructor() {
+    this.pool = new Pool({
+      // this may come from config module
+      user: 'postgres',
+      host: 'localhost',
```

Comment:
Agree! but since this is going to come from config in next PR, if you don't mind, I'll keep it like this and update in the next one as comment claims :p